secator/store_utils.py

```
@@ -0,0 +1,194 @@
+		self._watermark = high
+
+	def iter_results(self):
+		"""Generator: yield findings as they land; render the live panel from runner docs; stop when
+		the root doc is terminal or the inactivity window elapses."""
+		progress = self._make_panel() if self.print_remote_info else nullcontext()
+		last_advance = self._time()
+		with progress:
+			while True:
+				try:
+					runners = self._runners_in_scope()
+				except Exception as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)